jobs/curator/templates/bin/curator_ctl

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+set -u
+set -x
+
+CRON_SCHEDULE="*/2 * * * * /var/vcap/jobs/curator/bin/purge_logs"
+CRON_STATUS=`pgrep cron | wc -l`
+
+source /var/vcap/jobs/curator/helpers/ctl_setup.sh 'curator'
+
+case $1 in
+  start)
+    if [ $CRON_STATUS -eq 0 ]; then
+      /usr/sbin/cron start
```

Comment:
Just randomly found [this recent story about enabling cron on bosh-lite](https://www.pivotaltracker.com/story/show/101179464) while working on something else. Guess after that happens we can simplify this control script later.